data-plane/src/server/tls/cert_resolver.rs

```diff
@@ -133,15 +133,15 @@ impl AttestableCertResolver {
     //[cage-name].[cage-uuid].cage.evervault.[com|dev]
     pub(crate) fn is_trusted_cert_domain(received_servername: Option<&str>) -> bool {
         match received_servername {
-            Some(servername) => {
-                let split_hostname: Vec<&str> = servername.split('.').collect();
-                let subdomains_total = split_hostname.len();
-
-                split_hostname
-                    .get(subdomains_total - 3) //Second from the end
-                    .map(|specific_subdomain| specific_subdomain == &"cage")
-                    .unwrap_or(false)
-            }
+            Some(servername) => servername
+                .split('.')
+                .rev()
+                .nth(2)
+                .map_or(false, |subdomain| {
+                    println!("subdomain: {}", subdomain);
```

Comment:
Assume we don't need this? Can make it a `log::debug` either
```suggestion
```

Reply:
sorry every time 🤦